### PR TITLE
Rename MacOS application to "Endless Sky"

### LINKED ISSUE #561: Endless Sky has the name "EndlessSky" on OS X
On OS X, Endless Sky appears as "EndlessSky":

EndlessSky in the menu bar
EndlessSky in the dock

This is contrary to how it appears everywhere else (i.e. GitHub, Steam, the manual) where it is two separated words.

### PR DESCRIPTION
**Pull request by nobodywasishere**
- Changed the path of libSDL2-2.0.0.dylib to match build instructions
- removed (unused) localization 
- added Bundle Display Name and changed the Product Name

Fixes #561

### CODE REVIEW

File: EndlessSky.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -169,7 +169,7 @@
 /* End PBXCopyFilesBuildPhase section */
 
 /* Begin PBXFileReference section */
-		4C2DEF55201B8FAD0062315E /* libSDL2-2.0.0.dylib */ = {isa = PBXFileReference; lastKnownFileType = "compiled.mach-o.dylib"; name = "libSDL2-2.0.0.dylib"; path = "/usr/local/Cellar/sdl2/2.0.8/lib/libSDL2-2.0.0.dylib"; sourceTree = "<absolute>"; };
```
Comment (Pointedstick) [inline EndlessSky.xcodeproj/project.pbxproj]: This change does not seem intentional and may in fact result in breakage.

File: EndlessSky.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -823,7 +823,6 @@
 			developmentRegion = English;
 			hasScannedForEncodings = 0;
 			knownRegions = (
-				en,
```
Comment (Pointedstick) [inline EndlessSky.xcodeproj/project.pbxproj]: Is this change intentional?

File: XCode/en.lproj/InfoPlist.strings [hunk 3]
```diff
@@ -1,2 +0,0 @@
-/* Localized versions of Info.plist keys */
-
```
Comment (Pointedstick) [inline XCode/en.lproj/InfoPlist.strings]: Is deleting this file intentional?

File: EndlessSky.xcodeproj/project.pbxproj (hunk 1 above)
Comment (nobodywasishere) [reply EndlessSky.xcodeproj/project.pbxproj]: This is what I needed to change to get this to build in XCode on my computer. I installed through brew. This is the same location referenced in the Wiki for Build Instructions Library Paths. https://github.com/endless-sky/endless-sky/wiki/BuildInstructions

File: EndlessSky.xcodeproj/project.pbxproj (hunk 2 above)
Comment (nobodywasishere) [reply EndlessSky.xcodeproj/project.pbxproj]: Yes this change was intentional.

File: XCode/en.lproj/InfoPlist.strings (hunk 3 above)
Comment (nobodywasishere) [reply XCode/en.lproj/InfoPlist.strings]: Yes this change was intentional.

File: EndlessSky.xcodeproj/project.pbxproj (hunk 1 above)
Comment (Fzzr) [reply EndlessSky.xcodeproj/project.pbxproj]: @nobodywasishere please also test if you can create a fully distributable Mac build per those instructions. This is an issue we've been dealing with on the Discord. If you like, you may join us there at discord dot gg slash ZeuASSx.

**Comment by nobodywasishere** [comment]
I fixed an issue had with Travis CI not building due to spaces in the file path due to the name change. There are no other issues to address, unless I'm mistaken.

File: EndlessSky.xcodeproj/project.pbxproj (hunk 1 above)
Comment (nobodywasishere) [reply EndlessSky.xcodeproj/project.pbxproj]: Fully distributable MacOS build can be made still.
